src/container/srv_target.c

```diff
@@ -524,6 +525,16 @@ ds_cont_local_open(uuid_t pool_uuid, uuid_t cont_hdl_uuid, uuid_t cont_uuid,
 		*cont_hdl = hdl;
 	}
 
+	if (cont_uuid != NULL) {
+		rc = dtx_resync(hdl->sch_pool->spc_hdl, pool_uuid,
+				cont_uuid, hdl->sch_pool->spc_map_version);
+		if (rc != 0)
+			/* Just warning, not fail out. */
+			D_WARN("Fail to resync some DTX(s) status, that may "
+			       "cause some unexpected IO result in subsequent "
+			       "operations: rc = %d.\n", rc);
+	}
+
```

Comment:
Please add comments for why we need dtx_resync() during cont open.  And also could we do it asynchronously? since it is inside container open rpc handler.

Reply:
Yes, the comment has been added in former patch. We need to re-sync() the DTX status before accessing related objects. If the container open RPC returned, then means everything is ready.